LayoutTests/platform/wpe/TestExpectations

```diff
@@ -179,7 +179,6 @@ webkit.org/b/212083 loader/navigation-policy/should-open-external-urls/user-gest
 
 # Mediastream
 webkit.org/b/213202 fast/mediastream/getUserMedia-grant-persistency3.html [ Failure ]
-http/wpt/mediastream/mediastreamtrackprocessor-videoframe-timestamp.html [ Pass Slow Failure ]
```

Comment:
Looking up this test in results.webkit.org, shows the test is still flakey in WPEWebKit and WebKitGTK:

https://results.webkit.org/?suite=layout-tests&test=http%2Fwpt%2Fmediastream%2Fmediastreamtrackprocessor-videoframe-timestamp.html&platform=WPE&platform=GTK